cmd/perf-tests.go

```diff
@@ -338,3 +340,80 @@ func netperf(ctx context.Context, duration time.Duration) madmin.NetperfNodeResu
 	globalNetPerfRX.Reset()
 	return madmin.NetperfNodeResult{Endpoint: "", TX: r.n / uint64(duration.Seconds()), RX: uint64(rx / delta.Seconds()), Error: errStr}
 }
+
+func siteNetperf(ctx context.Context, duration time.Duration) madmin.SiteNetPerfNodeResult {
+	r := &netperfReader{eof: make(chan struct{})}
+	r.buf = make([]byte, 128*humanize.KiByte)
+	rand.Read(r.buf)
```

Comment:
You create this, but don't actually use it? Either there is something I don't understand, or this is not doing what you expect.